corehq/motech/repeaters/tasks.py

```diff
@@ -316,8 +316,26 @@ def process_repeaters():
 
     metrics_counter('commcare.repeaters.process_repeaters.start')
     try:
-        for repeater_id, lock_token in iter_repeater_id_tokens():
-            process_repeater(repeater_id, lock_token)
+        group = 0
+        client = get_redis_client()
+        client.set(f'repeater_group_{group}', 0)
```

Comment:
I think we want this key to expire at some point because we don't want to leave old/unused keys in redis indefinitely.